kraftwerk-core/src/main/java/fr/insee/kraftwerk/core/sequence/WriterSequence.java

```diff
@@ -8,11 +8,13 @@
 import fr.insee.kraftwerk.core.outputs.csv.CsvOutputFiles;
```

Comment:
https://stackoverflow.com/questions/3837959/how-to-have-the-formatter-wrap-code-with-intellij